### ISSUE TITLE
OverlayPanel: No aria attributes are added to trigger

### ISSUE THREAD

**Issue by puschie286** [primefaces/primeng#14290]
### Describe the bug

[Documentation](https://primeng.org/overlaypanel#accessibility) of overlaypanel describe trigger accessibility behaviour: "OverlayPanel adds aria-expanded state attribute and aria-controls to the trigger so that the relation between the trigger and the popup is defined."
this does not happen - no aria attribute is added to the trigger

### Environment

website

### Angular version

17

### PrimeNG version

17

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

not relevant

### Browser(s)

Firefox

### Steps to reproduce the behavior

Check any example of the OverlayPanel

### Expected behavior

1. aria-expanded and aria-controls is added to trigger on initialize
2. aria-expanded is updated on overlay show/hide

**Comment by cetincakiroglu** [comment]
Hi,

aria-expanded and aria-controls should be added into the trigger. In this case, with the overlay, trigger is not reside in the component itself, it's outer element that user configures. Because of that, user should add aria-expanded and aria-controls to the trigger element they use. 

We'll add ariaLabel and ariaLabelledBy properties in here https://github.com/primefaces/primeng/issues/14361

**Comment by puschie286** [comment]
@cetincakiroglu thanks for your response - i also think the cleanest way is if the developer add it. Then could you update the documentation to mention the responsebility of the developer to add these aria attributes ?
Additionaly, a notice about the [overlayVisibile](https://github.com/primefaces/primeng/blob/1ccac95cbf4f1fc86cc8f1feaa7654ab92107f02/src/app/components/overlaypanel/overlaypanel.ts#L179C10-L179C10) property would be great, i think it is the best candidate for using as aria expanded attribute link

**Comment by puschie286** [comment]
@cetincakiroglu
